test/Mock/MockEnum.php

```diff
@@ -3,7 +3,8 @@
 
 class MockEnum
 {
-    const FOO     = 1;
-    const BAR     = 2;
-    const FOO_BAR = 3;
+    const FOO       = 1;
+    const BAR       = 2;
+    const FOO_BAR   = 3;
+    const FOO_ARRAY = [1, 2, 3];
```

Comment:
Because this test fails in hhvm, can you add a new test for this specific case and mark it with `* @requires PHP 5.6`: https://phpunit.de/manual/current/en/incomplete-and-skipped-tests.html#incomplete-and-skipped-tests.requires.tables.api